plugins/backend/rag-ai-backend/src/service/types.ts

```diff
@@ -23,7 +23,7 @@ import { Config } from '@backstage/config';
 
 export interface RagAiConfig {
   logger: Logger | LoggerService;
-  tokenManager: TokenManager;
+  tokenManager?: TokenManager;
```

Comment:
Rather than making this a breaking change I'm making it optional. Maybe it should just be removed?